__tests__/app.test.js

```diff
@@ -41,6 +41,50 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api/articles", () => {
```

Comment:
this description is wrong its articles/:id

Reply:
I am organising the test suites in different describe blocks. On this instance, the describe block covers different methods for everything articles related (GET /api/articles, GET /api/articles/:article_id, GET /api/articles/:article_id/comments, etc.) Each one of those endpoints is another describe block which includes the tests for successful responses and errors. 
<img width="532" alt="Screenshot 2024-04-15 at 19 29 12" src="https://github.com/SaraGMatos/backend-project/assets/145334810/becba242-2ca1-40a9-b4fe-5c929996a12f">